frontend/app/(tabs)/create.tsx

```diff
@@ -13,42 +14,66 @@ import {
   H5,
   Text,
   TextArea,
-  View,
+  Switch,
 } from 'tamagui';
-import type { TabsContentProps } from 'tamagui';
 
 export default function Create() {
   const [isEnabled, setEnabledElements] = useState(false);
 
   return (
     <View>
       <YStack>
-        <H4 style={{ alignSelf: 'center' }}>NEW POST</H4>
-        <Tabs
-          defaultValue='tab1'
-          orientation='horizontal'
-          flexDirection='column'
-          width={400}
-          height={150}
-          borderRadius='$4'
-          borderWidth='$0.25'
-          overflow='hidden'
-          borderColor='$borderColor'
-        >
-          <Tabs.List
-            disablePassBorderRadius='bottom'
-            aria-label='Creating a New Post'
+        <H4 style={{ alignSelf: 'center', color: 'black' }}>NEW POST</H4>
+        <XStack>
+          <Text color={'black'} fontSize='$5' paddingStart='$5'>
+            is this a recipe?
+          </Text>
+          <Switch
+            marginLeft='$2'
+            size='$3'
+            onCheckedChange={() => setEnabledElements(!isEnabled)}
           >
-            <Tabs.Tab flex={1} value='tab1'>
-              <SizableText fontFamily='$body'>Byte</SizableText>
-            </Tabs.Tab>
-            <Tabs.Tab flex={1} value='tab2'>
-              <SizableText fontFamily='$body'>Recipe</SizableText>
-            </Tabs.Tab>
-          </Tabs.List>
-          <Tabs.Content value='tab1'>
+            <Switch.Thumb animation='bouncy' />
+          </Switch>
+        </XStack>
+        {!isEnabled ? (
+          <YStack>
+            <TextArea
```

Comment:
Same as above.

We should utilize the `UserInput` component that I have written that validates user input before it is sent to the backend. Check my code in the files `login.tsx` or `register.tsx`. You will also need to add the create a `Zod Schema` for this. Check the `user.ts` file for reference. Make a **new**  ts file for this.

With this, you will also have to use the `form` from tamagui and some others but you can follow the `login.tsx` or `register.tsx` to figure it out. A lot of the code will be similar.